### PR TITLE
Wasm compiler service

### PR DESCRIPTION
**Pull request by osipov-mit**
- To compile rust project it necessary to make post request to `/wasm-compiler/build` endpoint with parameter `file` and its value have to be buffer from archived project in zip. This request returns id of compiling process.
- To get result of compiling process it necessary to make post request to `/wasm-compiler/get-wasm` endpoint with parameter `id` which have to be equal the value received on the previous step

### CODE REVIEW

File: website/wasm-compiler/src/util.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const AdmZip = require('adm-zip');
+import { mkdirSync } from 'fs';
+import { join } from 'path';
+import config from './config/configuration';
+
+export function isWasm(fileName: string): boolean {
+  let ext = fileName.split('.');
+  return ext[ext.length - 1] === 'wasm';
```
Comment (codev0) [inline website/wasm-compiler/src/util.ts:8]: I think its weak way to check file kind, maybe checking mime type is better?

File: website/wasm-compiler/src/util.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+export function generateId(): string {
+  return Math.random().toString(36).substring(2);
```
Comment (codev0) [inline website/wasm-compiler/src/util.ts:12]: wow, it's nice

Comment (osipov-mit) [reply website/wasm-compiler/src/util.ts:12]: 😊

File: website/wasm-compiler/src/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  imports: [
+    ConfigModule.forRoot({
+      load: [configuration],
+    }),
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: (configService: ConfigService) => ({
+        type: 'postgres',
+        host: configService.get('db.host'),
+        port: +configService.get<number>('db.port'),
```
Comment (codev0) [inline website/wasm-compiler/src/app.module.ts:19]: I think casting to number via parseInt more reliable

**Review by codev0 — APPROVED** [review]

**Pull request merged**